docs/connector.md

```diff
@@ -0,0 +1,51 @@
+# Connector
+
+The [`backend`](./architecture.md#backend) includes an [Interledger](./glossary.md#interledger-protocol) connector for sending and receiving [STREAM](./glossary.md#stream) packets.
+
+The amounts of these packets are used to update account balances in [TigerBeetle](./glossary.md#tigerbeetle). (See [Accounts and Transfers](./accounts-and-transfers.md))
```

Comment:
I think we have used "Tigerbeetle" rather than "TigerBeetle" within the docs. I think we should make that consistent.

Reply:
I vote we match their capitalization